webpack/config.prod.standalone.js

```diff
@@ -0,0 +1,156 @@
+const webpack = require('webpack');
+const path = require('path');
+const TerserJSPlugin = require('terser-webpack-plugin');
+const CopyWebpackPlugin = require('copy-webpack-plugin');
+const OptimizeCSSAssetsPlugin = require('optimize-css-assets-webpack-plugin');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const postcssPresetEnv = require('postcss-preset-env');
+const postcssFocus = require('postcss-focus');
+
+module.exports = {
+  mode: 'production',
+  devtool: 'source-map',
+  target: 'web',
+  entry: {
+    p5: './index.js'
```

Comment:
Since we can select the entry file here, wouldn't it be easier to create a different index.js file with our changes while maintaining the original file (and facilitate merging back from forked package)?